### PR TITLE
Fix typos and update referenced Kitura version

### PR DESCRIPTION
**Pull request by helenmasters**

### CODE REVIEW

File: en/resources/tutorials/templating.md [hunk 1]
```diff
@@ -41,9 +41,14 @@ import PackageDescription
 let package = Package(
     name: "myTemplates",
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", .upToNextMinor(from: "1.0.0")),
-        .package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", .upToNextMinor(from: "1.0.0"))
+        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.2.0")),
```
Comment (seabaylea) [inline en/resources/tutorials/templating.md]: For it to be possible to cut/paste this as a Package.swift file, the very first line needs to be:
```
// swift-tools-version:4.0
```

File: en/resources/tutorials/templating.md [hunk 2]
```diff
@@ -41,9 +41,14 @@ import PackageDescription
...
+        .package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", .upToNextMajor(from: "1.0.0")),
+        .package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", .upToNextMajor(from: "1.0.0"))
+		]
```
Comment (seabaylea) [inline en/resources/tutorials/templating.md]: missing comma after the close bracket (i.e. `],`)

File: en/resources/tutorials/templating.md [hunk 3]
```diff
@@ -41,9 +41,14 @@ import PackageDescription
...
+		targets: [
+		    .target(name: "myTemplates", dependencies: [ .target(name: "Application"), "Kitura" , "HeliumLogger"]),
+		    .target(name: "Application", dependencies: [ "Kitura", "KituraStencil", "KituraMustache" ])
+		]
  ])
```
Comment (seabaylea) [inline en/resources/tutorials/templating.md]: Extra closing square bracket that needs to be removed

File: en/resources/tutorials/templating.md [hunk 4]
```diff
@@ -16,16 +16,16 @@ redirect_from: "/resources/templating.html"
...
+Why use a template engine in your application? A [template engine](https://en.wikipedia.org/wiki/Template_processor) allows rendering of documents using static templates by substituting template variables with actual values at runtime. This means that you have clear separation between your template and data. You can also generate multiple web pages dynamically with a single template! This page explains how you can use template engines integrated with Kitura in your Kitura apps.
 
 ---
 
 ## Kitura template engines
-Kitura template engines are classes that implement the `TemplateEngine` protocol from the [Kitura-TemplateEngine package](https://github.com/IBM-Swift/Kitura-TemplateEngine/blob/master/Sources/KituraTemplateEngine/TemplateEngine.swift). Currently, three Kitura template engines are supported:
+Kitura template engines are classes that implement the `TemplateEngine` protocol from the [Kitura-TemplateEngine package](https://github.com/IBM-Swift/Kitura-TemplateEngine/blob/master/Sources/KituraTemplateEngine/TemplateEngine.swift). The most commonly used template engine is Stencil. Currently, three Kitura template engines are supported:
```
Comment (seabaylea) [inline en/resources/tutorials/templating.md]: It feels like a more natural reading would be to reverse the sentence, i.e.:
"Kitura currently supports three template engines, with Stencil being the most commonly used:"

File: en/resources/tutorials/templating.md (hunk 1 above)
Comment (helenmasters) [reply en/resources/tutorials/templating.md]: I've added this as you suggested.

File: en/resources/tutorials/templating.md (hunk 2 above)
Comment (helenmasters) [reply en/resources/tutorials/templating.md]: Good spot, fixed.

File: en/resources/tutorials/templating.md (hunk 3 above)
Comment (helenmasters) [reply en/resources/tutorials/templating.md]: Another good spot, now fixed.

File: en/resources/tutorials/templating.md (hunk 4 above)
Comment (helenmasters) [reply en/resources/tutorials/templating.md]: Reworded as it does sound a lot better that way round.

**Pull request merged**